2021/22xxx/CVE-2021-22564.json

```diff
@@ -1,18 +1,90 @@
 {
```

Comment:
File name CVE ID and JSON CVE ID do not match.

Reply:
Fixed.